src/index.html

```diff
@@ -10,23 +10,34 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
-    <title>Document</title>
+    <title>Search Bar</title>
     <link
       rel="stylesheet"
       href="style.css"
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      class="search-bar search-bar-top"
+      data-qa="big"
+    >
+      <input
+        class="search-bar-input search-bar-input--top"
+        type="text"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+      />
```

Comment:
You have two forms here that seem to be doing the same thing. Unless they have different purposes, you might want to consider combining them into one form to avoid repetition. This will make your code more DRY (Don't Repeat Yourself), which is a key principle in software development.